src/chapters/sql-part1/studio.rst

```diff
@@ -0,0 +1,269 @@
+the movie, the ``year`` it was released, and the ``director``. We're also going
+to want to have a unique ``movie_id`` column as the primary key, since there
+can be more than one movie with the same title.
+
+Take the time now to think about what data types you might use for these
+columns (``INTEGER``, ``VARCHAR``, ``DATE``, etc.), then go ahead and write (or
+type) the SQL statement to create this table.
+
+As you consider which data types to use for your ``movies`` columns, you might
+find yourself thinking, *If this were a property of a Java object, what data
+type would I use?*
+
+For example, if the ``movie`` object has a ``title`` and a ``director`` field,
+you would expect both of these data types to be ``String``. Similarly the year
+the movie was released would be stored as an integer. This conceptual overlap
+between databases and Java objects is very useful, and we will explore this
+further in the class on *Object-Relational Mapping* (ORM).
+
+Here's how our ``movies`` table could be created:
+
+.. sourcecode:: sql
+   :linenos:
+
+   CREATE TABLE movies (
+      movie_id INTEGER PRIMARY KEY AUTO_INCREMENT,
+      title VARCHAR(120),
+      year INTEGER,
+      director VARCHAR(80)
+   );
+
+(Take a moment to discuss with a partner why the two ``VARCHAR`` parameters
+differ).
+
+Directors Table
+^^^^^^^^^^^^^^^
+
+Next, let's consider the properties we'd want for a ``director`` object in
+Java. These could include a ``first`` name, a ``last`` name, and maybe the
+``country`` where the director was born.
+
+Once we identify the data types for these properties, we can write the SQL
+command to create the ``directors`` table. Also, we should include a
+``director_id`` column in order to create a valid primary key.
+
+.. _directors-table:
+
+.. sourcecode:: sql
+   :linenos:
+
+   CREATE TABLE directors (
+      director_id INTEGER PRIMARY KEY AUTO_INCREMENT,
+      first VARCHAR(40),
+      last VARCHAR(40),
+      country VARCHAR(80)
+   );
+
+Relating the Tables
+^^^^^^^^^^^^^^^^^^^^
+
+Before we go further, let's take a moment to think about how these two tables
+relate to each other, and whether we may want to modify the column ``director``
+in the ``movies`` table. Understanding the common *one-to-many* relationship
+that exists between database tables helps us set up the best design for
+``movies`` and ``directors``. A director can work on multiple movies, but each
+movie may only have one director.
+
+.. admonition:: Note
+
+   It may be useful to review `this article <http://www.databaseprimer.com/pages/relationship_1tox/>`__
+   to refresh your memory on the one-to-many concept.
+
+In order to reflect a one-to-many relationship in the table definitions, let's
+modify the ``director`` column in ``movies`` to become ``director_id``. It
+should therefore be an ``INTEGER`` instead of the ``VARCHAR`` we used earlier
+when we thought the ``director`` column would hold the name of a person.
```

Comment:
They won't learn one-to-many until the next chapter.

Reply:
Maybe, just go over how a director can direct many movies and to connect the two tables, we need to add a ``director_id``?

Reply:
I re-rewrote this section to try to address this.  Let me know what you think.